uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/services/assignment.js

```diff
@@ -1,10 +1,12 @@
-/* globals angular, CRM */
-/* eslint-env amd */
+/* eslint-env amd, jasmine */
 
 define([
+  'common/angular',
+  'common/lodash',
+  'tasks-assignments/services/contact',
   'tasks-assignments/services/services',
```

Comment:
```js
define([
  'common/angular',
  'common/lodash',
  'tasks-assignments/services/services',
  'tasks-assignments/services/contact',
  'tasks-assignments/services/utils'
], function (angular, _, services) {
});
```
you need to require `services/contact` but you don't need it as the callback parameter